File: push/src/main/java/com/xiaomi/mipush/sdk/ManifestCheckerAspectLog.java
package com.xiaomi.mipush.sdk;

import android.content.pm.PackageInfo;

import com.elvishew.xlog.Logger;
import com.elvishew.xlog.XLog;

import org.aspectj.lang.JoinPoint;

public class ManifestCheckerAspectLog {
    private static final String TAG = ManifestCheckerAspect.class.getSimpleName();
    private static final Logger logger = XLog.tag(TAG).build();

    public void logCheckServices(final JoinPoint joinPoint, PackageInfo pkgInfo) {
        logger.d(joinPoint.getSignature());
        logger.d(pkgInfo);
    }
}

File: push/src/main/java/com/xiaomi/mipush/sdk/ManifestCheckerAspectLog.kt
package com.xiaomi.mipush.sdk

import android.content.pm.PackageInfo
import com.elvishew.xlog.Logger
import com.elvishew.xlog.XLog
import org.aspectj.lang.JoinPoint

class ManifestCheckerAspectLog {
    fun logCheckServices(joinPoint: JoinPoint, pkgInfo: PackageInfo) {
        logger.d(joinPoint.signature)
        logger.d(pkgInfo)
    }

    companion object {
        private val TAG: String = ManifestCheckerAspect::class.java.simpleName
        private val logger: Logger = XLog.tag(TAG).build()
    }
}
